### DISCUSSION TITLE
邮箱收不到授权码，也无法升级到专业版

### DISCUSSION THREAD

**Discussion by ruoruosun** [getlantern/lantern-forum-cn#835] [General]
应该是代理处购入的串码，现在无法收到邮箱授权认证
授权邮箱346424925@qq.com，已经发送过账单截图，请尽快处理

**Comment by MaxMKMax** [comment]
@ruoruosun 您好请问您的支付单据发到哪一个邮箱? 您方便发到这里吗? 谢谢.
